research/eccv2024/download_raw_images.py

```diff
@@ -0,0 +1,29 @@
+"""
+Author: Aditya Jain
+Date last modified: October 25, 2023
+About: Download raw images for the annotated tasks
+"""
+
+import json
+import urllib.request
+import os
+
+annotation_file = (
+    "/home/mila/a/aditya.jain/scratch/cvpr2024_data/annotated-tasks-20231106.json"
+)
+data = json.load(open(annotation_file))
+image_folder = "/home/mila/a/aditya.jain/scratch/cvpr2024_data/ami_traps_dataset/images"
```

Comment:
Make this a relative path, consider using an environment variable, or a command line option